cmd/printverifier/main.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2024 The Armored Witness authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// The printverifier command prints a note compatible verifier string for
+// a public key stored in GCP.
+package main
+
+import (
+	"context"
+	"crypto/ed25519"
+	"crypto/x509"
+	"encoding/pem"
+	"flag"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	kms "cloud.google.com/go/kms/apiv1"
+	"cloud.google.com/go/kms/apiv1/kmspb"
+	"golang.org/x/mod/sumdb/note"
+)
+
+var (
+	keyResource = flag.String("key", "", "GCP Resource ID for the public key to use")
+	keyName     = flag.String("name", "", "Template for verifier name, use %%d to include keyRevision")
```

Comment:
Should we clarify that this is the note key name? May be confusing since the keyResource above is a GCP "name".

Reply:
Good call - PTAL